donate.md

```diff
@@ -23,6 +23,8 @@ All Hands Active depends on your generosity. Any donation helps, but we are espe
 <input type="image" src="https://www.paypalobjects.com/en_US/i/btn/btn_donateCC_LG.gif" border="0" name="submit" title="PayPal - The safer, easier way to pay online!" alt="Donate with PayPal button" />
 <img alt="" border="0" src="https://www.paypal.com/en_US/i/scr/pixel.gif" width="1" height="1" />
 </form>
+<br>
+Additionally, we have a wish-list that can be found <a href="https://docs.google.com/spreadsheets/d/11-hJSy9P6lK2bZz6fs8hZr2WjJZGcyNai22qJ6eJq8k/edit#gid=2110100171">here</a>, and additional items can be submitted <a href="https://docs.google.com/forms/d/e/1FAIpQLSeGz1hoymTJDEhyTOof_Smwutxoh1JhQ7nkZ8P_UcQWpAOrxg/viewform">here</a>.
```

Comment:
Can you please use markdown here? Also could you avoid links with non-descriptive link text - for example linking the phrase "wish list" is far more accessible than the word "here".